packages/contracts/contracts/tokens/ERC20Token.sol

```diff
@@ -0,0 +1,68 @@
+pragma solidity ^0.4.18;
```

Comment:
Why did we get rid of UnlimitedAllowance contract?

Reply:
Added it back in.